tests/cases/integration/types.mysql.phpt

```diff
@@ -89,16 +89,19 @@ class TypesMysqlTest extends IntegrationTestCase
 	{
 		$this->connection->query("
 			CREATE TEMPORARY TABLE [types_write] (
-				[blob] blob
+				[blob] blob,
+				[json] text
```

Comment:
I forgot they have added it in 5.7, but it seems they don't support it directly https://www.google.cz/search?q=mysql+5.7+travis yet.